lib/directory-view.coffee

```diff
@@ -27,10 +27,11 @@ class DirectoryView extends HTMLElement
       iconClass = 'icon-file-symlink-directory'
     else
       iconClass = 'icon-file-directory'
-      if @directory.isRoot
-        iconClass = 'icon-repo' if repoForPath(@directory.path)?.isProjectAtRoot()
+      relPath = repoForPath(@directory.path)?.relativize(@directory.path + '/')?.length
+      if relPath == 0
```

Comment:
unless triggers in case of 'undefined' as well, but in this case it should trigger only if string is empty(but not undefined).